index.html

```diff
@@ -247,6 +247,54 @@
         particular, the algorithms defined in this specification are intended
         to be easy to follow, and not intended to be performant.)
       </p>
+      <section>
+        <h3>
+          Conformance Classes
+        </h3>
+        <p>
+          This specification describes the conformance criteria for two classes
+          of user agents.
+        </p>
+        <dl>
+          <dt>
+            <dfn>Controlling user agent</dfn>
+          </dt>
+          <dd>
+            <p>
+              Web browsers that conform to the specifications of a
+              <a>controlling user agent</a> must be able to start and control
+              presentations by providing a <a>controlling browsing context</a>
+              as described in this specification. This context MUST implement
+              the <a><code>PresentationConnection</code></a>,
+              <a><code>PresentationConnectionAvailableEvent</code></a>, and
+              <a><code>PresentationRequest</code></a> interfaces.
+            </p>
+          </dd>
+          <dt>
+            <dfn>Receiving user agent</dfn>
+          </dt>
+          <dd>
+            <p>
+              Web browsers that conform to the specifications of a <a>receiving
+              user agent</a> must be able to render presentations by providing
+              a <a>receiving browsing context</a> as described in this
+              specification. This context must implement the
+              <a><code>PresentationConnection</code></a>,
+              <a><code>PresentationConnectionAvailableEvent</code></a>, and
+              <a><code>PresentationReceiver</code></a> interfaces.
+            </p>
```

Comment:
Same comments as above: s/This context must implement/A receiving user agent implements/ and add "Presentation" interface as well.